ui/src/app/cohort-search/list-overview/list-overview.component.tsx

```diff
@@ -0,0 +1,426 @@
+                </Clickable>
+              </TooltipTrigger>
+              <TooltipTrigger content={<div>Delete cohort</div>}>
+                <Clickable style={{...styles.actionIcon, ...(disableIcon ? styles.disabled : {})}}
+                  onClick={() => this.setState({deleting: true})}>
+                  <ClrIcon shape='trash' className='is-solid' size={30} />
+                </Clickable>
+              </TooltipTrigger>
+              <TooltipTrigger content={<div>Review participant level data</div>}>
+                <Clickable style={{...styles.actionIcon, ...(disableIcon ? styles.disabled : {})}}
+                  onClick={() => this.navigateTo('review')}>
+                  <ClrIcon shape='copy' className='is-solid' size={30} />
+                </Clickable>
+              </TooltipTrigger>
+            </div>
+            <h2 style={styles.totalCount}>
+              Total Count: &nbsp;
+              {loading && !this.hasTemporalError && <Spinner size={18} />}
+              {!loading && !this.hasErrors && total !== undefined &&
+                <span>{total.toLocaleString()}</span>
+              }
+              {this.hasErrors && <span>
+                -- <TooltipTrigger content={this.hasTemporalError ?
+                    'Please complete criteria selections before saving temporal relationship.' :
+                    `All criteria are suppressed. Un-suppress criteria to update the total count
+                     based on the visible criteria.`}>
+                  <ClrIcon style={{color: '#F57600'}} shape='warning-standard' size={18} />
+                </TooltipTrigger>
+              </span>}
+            </h2>
+          </div>
+          {error && !this.hasErrors && <div style={styles.totalError}>
+            <ClrIcon className='is-solid' shape='exclamation-triangle' size={22} />
+            Sorry, the request cannot be completed.
+          </div>}
+          {!this.hasErrors && !loading && !!chartData && total && <div style={styles.cardContainer}>
+            <div style={styles.card}>
+              <div style={styles.cardHeader}>
+                Results by Gender
+              </div>
+              <div style={{padding: '0.5rem 0.75rem'}}>
+                {chartData.size && <GenderChart data={chartData} />}
+              </div>
+            </div>
+            <div style={styles.card}>
+              <div style={styles.cardHeader}>
+                Results By Gender, Age Range, and Race
+                <ClrIcon shape='sort-by'
+                  className={stackChart ? 'is-info' : ''}
+                  onClick={() => this.toggleChartMode()} />
+              </div>
+              <div style={{padding: '0.5rem 0.75rem'}}>
+                {chartData.size &&
+                  <ComboChart mode={stackChart ? 'stacked' : 'normalized'} data={chartData} />}
+              </div>
+            </div>
+          </div>}
+        </div>
+        {saveModal && <Modal>
```

Comment:
nit: i would rename this state piece to "saveModalOpen" or something that better conveys what it is